public/html/employee.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
```

Comment:
I like this look, but can you convert it over to handlebars? I can help work on this in class today.

Reply:
These html's should have been blank.  I used them for quicker testing but moved the code when I was done.  The handlebar files are further down below